docs/usage/help.rst.inc

```diff
@@ -58,8 +58,16 @@ Regular expressions, selector `re:`
     the re module <https://docs.python.org/3/library/re.html>`_.
 
 Path prefix, selector `pp:`
-    This pattern style is useful to match whole sub-directories. The pattern
-    `pp:root/somedir` matches `root/somedir` and everything therein.
+    This pattern style is useful to match whole sub-directories without worrying
+    about escaping special characters. Otherwise, you can just use the default
```

Comment:
I'ld rather remove the "not worrying" here and add a "be careful to escape special chars so borg gets to see them and they are not expanded by your shell" to the patterns that frequently have a `*` or so in them.

Reply:
(please check if we already have a warning about shell wildcard expansion and quoting somewhere)

Reply:
How about, "This pattern style is useful for matching whole sub-directories without having to escape special characters (beyond any escaping needed when using it within a shell)".

> (please check if we already have a warning about shell wildcard expansion and quoting somewhere)

Yeah, there's a line later on that says "When used from within a shell the patterns should be quoted to protect them from expansion."

Reply:
ok, that should be enough. no change needed.

it should be clear that if you don't use wildcards, that you don't need to quote any, of course.